notebooks/Imputation_best_practices/Imputation_best_practices.ipynb

```diff
@@ -0,0 +1,4557 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    for i in number_1_idx:
</pre><p>Is it possible to do these computations in vector form instead of a for loop? Maybe something like this?</p><pre class="ql-syntax" spellcheck="false">number_diff_1 = (imputed_number_df['number_copy_1_percent'] - imputed_number_df['number']).abs()
number_diff_1 = number_diff_1.loc[number_1_idx]
</pre><p>( I'm just eyeballing a potential solution, haven't run it to check if it's 100% accurate :) )</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/data-science-workflows/pull/113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>For loop is more convenient, the idea here is to store the indexes of the randomly creating missing values and then perform imputation only on those.Vector form will calculate the imputation and difference in all.</p>